content/projects/gsoc/2024/project-ideas/buiilding-ios-app.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: gsocprojectidea
+title: "Building iOS Apps with Jenkins"
+goal: "Describe best practices and provide architectural templates for building iOS applications with Jenkins"
+category: Tools
+year: 2024
+status: draft
+sig: platform
+skills:
+- Java
+- YAML
+- iOS development
+- Command line tools
+- Package management tool theory
+mentors:
+- "gounthar"
+links:
+    emailThread: https://community.jenkins.io/t/gsoc-2023-project-idea-building-ios-apps-with-jenkins/4799
+---
+//   gitter: "jenkinsci_plugin-installation-manager-cli-tool:gitter.im"
+//   draft: https://docs.google.com/document/d/1s-dLUfU1OK-88bCj-GKaNuFfJQlQNLTWtacKkVMVmHc
+
+=== Background
+
+Applying CI principles when building iOS applications brings its own challenges, different from those of more classical applications.
+How to proceed, what are the successful patterns, what are the pitfalls is poorly documented. 
+It very often requires potential user and Jenkins Administrator to "reinvent the wheel".
```

Comment:
```suggestion
Topics such as _how to proceed_, _what are the successful patterns_, and _what are the pitfalls_ are poorly documented. 
It very often requires potential users and Jenkins Administrators to "reinvent the wheel".
```
suggestion to add some syntax so that the sentence reads more clearly and emphasizes the topics a bit more, also adds pluralization for users and administrators